src/fancy/core/Grid9.hx

```diff
@@ -312,8 +312,11 @@ class Grid9 {
   }
 
   public function sizeRails(topRail: Float, bottomRail: Float, leftRail: Float, rightRail: Float) {
-    if(this.topRail == topRail && this.bottomRail == bottomRail && this.leftRail == leftRail && this.rightRail == rightRail)
-      return;
+    // FIXME: i commented out the early return because sometimes this function
```

Comment:
This is the only function that updates the `contentHeight` and `contentWidth` styles in the DOM, but the guard was causing it to return early if rails hadn't changed. But in a case where rails hadn't changed but contentheight/width had, i still want to hit all of the DOM updates below. Really, we should probably split this whole function into two.